syntax = "proto3";

message Person {
  string id = 1;
  string name = 2;
  string gender = 3;
  string age = 4;
  string eye_color = 5;
  string hair_color = 6;
  repeated string films = 7;
  string species = 8;
  string url = 9;
}

message PeopleList {
  repeated Person people = 1;
}
